### ISSUE TITLE
Did anyone reproduce TSN on ActivityNet v1.3? Or are there any pretrained models on ActivityNet v1.3 available?

### ISSUE THREAD

**Issue by raxxerwan** [yjxiong/tsn-pytorch#106]
Hi everyone,
I implemented a pytorch version of TSN, and the performence of my TSN on UCF101 is perfectly fine.

However, according to the original paper ([https://arxiv.org/pdf/1705.02953.pdf](TSN)), the author mentioned that TSN gets mAP 79.4% on the validation set of ActivityNet v1.3. I strictly follow the training settings in the paper to train my TSN on ActivityNet v1.3 and the testing methods (M-TWI) proposed in the paper, but I can only get mAP around 73%-74%.

Apparently, there must be something wrong in my implement. Did anyone reproduce TSN on ActivityNet v1.3? Or are there any pretrained models on ActivityNet v1.3 available? Any clues would be greatly appreciated. Thanks!
